src/win/getaddrinfo.c

```diff
@@ -380,3 +382,28 @@ int uv_getaddrinfo(uv_loop_t* loop,
   }
   return uv_translate_sys_error(err);
 }
+
+int uv_if_indextoname(unsigned int ifindex, char* ifname, size_t *sizep) {
+  NET_LUID InterfaceLuid;
+  char InterfaceName[NDIS_IF_MAX_STRING_SIZE + 1];
+  int required_len;
+  int ifname_len;
+
+  ifname_len = *sizep;
+
+  if (!ConvertInterfaceIndexToLuid(ifindex, &InterfaceLuid))
+    return uv_translate_sys_error(GetLastError());
+
+  if (!ConvertInterfaceLuidToAName(
+      InterfaceLuid, InterfaceName, sizeof(InterfaceName)))
```

Comment:
Some issues here:

- s/ConvertInterfaceLuidToAName/ConvertInterfaceLuidToNameA
- It should be `&InterfaceLuid`.
- It returns `zero` on success